beamer/tests/util.py

```diff
@@ -270,9 +270,9 @@ class CommandFailed(Exception):
     pass
 
 
-def run_command(*args):
+def run_command(cmd, *args):
     runner = CliRunner()
-    result = runner.invoke(*args)
+    result = runner.invoke(cmd, args, catch_exceptions=False)
```

Comment:
why catch_exceptions=False ?

Reply:
Part of the previously merged batch, but I want to answer anyway: this is so that exceptions propagate to pytest/ape and are visible immediately when running tests, instead of us having to run PDB when `CommandFailed` is raised and inspecting `result.exc_info` manually.